packages/react-query/src/useSuspenseQuery.ts

```diff
@@ -0,0 +1,30 @@
+'use client'
+import { QueryObserver } from '@tanstack/query-core'
+import { useBaseQuery } from './useBaseQuery'
+import type { UseQueryOptions } from './types'
+import type { DefaultError, QueryClient, QueryKey } from '@tanstack/query-core'
+import type { DefinedUseQueryResult } from './types'
+
+export function useSuspenseQuery<
+  TQueryFnData = unknown,
+  TError = DefaultError,
+  TData = TQueryFnData,
+  TQueryKey extends QueryKey = QueryKey,
+>(
+  options: Omit<
+    UseQueryOptions<TQueryFnData, TError, TData, TQueryKey>,
+    'enabled' | 'suspense' | 'throwOnError' | 'placeholderData'
+  >,
+  queryClient?: QueryClient,
+): Omit<DefinedUseQueryResult<TData, TError>, 'isPlaceholderData'> {
+  return useBaseQuery(
+    {
+      ...options,
+      enabled: true,
```

Comment:
```suggestion
```

Could I know why you forced `enabled: true` without exposing `enabled`? I thought enabled also could be exposed. I'd like to show you how exposing `enabled` in @suspensive/react-query. 
like this code https://github.com/suspensive/react/blob/main/packages/react-query/src/useSuspenseQuery.ts

Is it too many type overriding function?

If we force `enabled: true`, existing useQuery users can fetch according to the `enabled` option, but if it is a removed feature, it will be regrettable when using useSuspenseQuery. What's your Idea of removing?

Reply:
The main reason is that with suspense, we'd expect `data` to be defined, which is not the case if you disable a Query. We could add a type overload for `enabled`, but I'm not sure if it's needed (or if it's possible if we switch to a react specific cache implementation).

we can always add `enabled` back again in a minor if necessary, but I'd like to start out with the most basic version.

Reply:
> we can always add enabled back again in a minor if necessary, but I'd like to start out with the most basic version.

Cool, You mean we want to avoid unnecessary BREAKING CHANGE. I understood your intention of why you didn't expose enabled right now.